### PR TITLE
fix(macOS): contentControl intercepts pointer events

### LINKED ISSUE #6418: [MacOS] ContentControl with ContentControl intercepts pointer events
## Current behavior

When a ContentControl with a ContentTemplate is in a view the user is unable to interact with any of the controls.

## Expected behavior

The UI should work normally.

## How to reproduce it (as minimally and precisely as possible)
(Attached is the following project already set up)
[TestUnoMacApp.macOS.zip](https://github.com/unoplatform/uno/files/6779645/TestUnoMacApp.macOS.zip)

- Create a blank Uno project
- In MainPage.xaml put this:

```
    <Grid>
        <Button Grid.Row="1" Content="Click Me"/>
        <ContentControl
            Grid.Row="2"
            ContentTemplate="{StaticResource UserNameTemplate}" />
    </Grid>
```

- In a file called "UserNameTemplates.xaml" include the data template like:

```
    <DataTemplate x:Key="UserNameTemplate">
        <TextBlock Text="User Name Here"/>
    </DataTemplate>
```

- You will see that the template is visible on screen with the button, but the button does not respond to the mouse clicks.

## Workaround
none

## Works on UWP/WinUI
Yes

## Environment

Nuget Package:

- [X] Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia
- [ ] Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia
- [ ] Uno.SourceGenerationTasks
- [ ] Uno.UI.RemoteControl / Uno.WinUI.RemoteControl
- [ ] Other: 

Nuget Package Version(s):
Uno.Ui (3.9.0-dev-108)
Uno.UI.RemoteControl (same)

Affected platform(s):

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [X] macOS
- [ ] Skia
  - [ ] WPF
  - [ ] GTK (Linux)
  - [ ] Tizen
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

IDE:

- [ ] Visual Studio 2017 (version: )
- [ ] Visual Studio 2019 (version: )
- [X] Visual Studio for Mac (version: 8.10.4 (build 11))
- [ ] Rider Windows (version: )
- [ ] Rider macOS (version: )
- [ ] Visual Studio Code (version: )

Relevant plugins:

- [ ] Resharper (version: )

### PR DESCRIPTION
**Pull request by ajpinedam**
GitHub Issue (If applicable): #
closes #6418 

## PR Type

What kind of change does this PR introduce?
- Bugfix

## What is the current behavior?
ContentControl in a Window is intercepting the pointer events

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Review by jeromelaban — APPROVED** [review]

**Comment by MartinZikmund** [comment]
@ajpinedam Does this also resolve https://github.com/unoplatform/uno/issues/6146 ?

**Review by MartinZikmund — APPROVED** [review]

File: src/Uno.UI/UI/Xaml/UIElement.macOS.cs [hunk 1]
```diff
@@ -109,6 +109,12 @@ internal Windows.Foundation.Point GetPosition(Point position, global::Windows.UI
 		public override bool AcceptsFirstResponder()
 			=> true; // This is required to receive the KeyDown / KeyUp. Note: Key events are then bubble in managed.
 
+
+		internal static void LoadingRootElement(UIElement visualTreeRoot) { }
```
Comment (davidjohnoliver) [inline src/Uno.UI/UI/Xaml/UIElement.macOS.cs:113]: Is this a placeholder for future use?

Comment (jeromelaban) [reply src/Uno.UI/UI/Xaml/UIElement.macOS.cs:113]: Likely. We can keep it as a reminder that it's placed at the right location.

Comment (ajpinedam) [reply src/Uno.UI/UI/Xaml/UIElement.macOS.cs:113]: Yes @davidjohnoliver indeed, this is some sort of placeholder.

**Pull request merged**